src/protobuf/rpc.proto

```diff
@@ -15,3 +18,23 @@ enum CheckVersionStatus {
 message CheckVersionRequest { string version_str = 1; }
 
 message CheckVersionReply { CheckVersionStatus status = 1; }
+
+// Operation result of each atomic action
+enum OperationStatus {
+  SUCCESS = 0;
+  FAIL = 1;
+}
+
+message CreateTableRequest {
+  string table_name = 1;
+  repeated Schema fields = 2;
+}
+
+message Schema {
```

Comment:
It could be refined as:
```proto
enum FieldType {
  INT = 0;
  STRING = 1;
  DOUBLE = 2;
  // ...
}

message Filed {
  string name = 0;
  FieldType type = 1;
}

message TableScema {
  string table_name = 0;
  repeated Filed all_fields = 1;
}
```